extensions/mssql/src/main.ts

```diff
@@ -87,6 +87,29 @@ export async function activate(context: vscode.ExtensionContext): Promise<IExten
 		}
 	});
 
+	vscode.commands.registerCommand(Constants.cmdObjectExplorerEnableGroupBySchemaCommand, async () => {
+		await vscode.workspace.getConfiguration().update(Constants.configObjectExplorerGroupBySchemaFlagName, true, true);
+	});
+
+	vscode.commands.registerCommand(Constants.cmdObjectExplorerDisableGroupBySchemaCommand, async () => {
+		await vscode.workspace.getConfiguration().update(Constants.configObjectExplorerGroupBySchemaFlagName, false, true);
+	});
+
+	vscode.workspace.onDidChangeConfiguration(async e => {
+		if (e.affectsConfiguration(Constants.configObjectExplorerGroupBySchemaFlagName)) {
+			const groupBySchemaTelemetryActionEvent = vscode.workspace.getConfiguration().get(Constants.configObjectExplorerGroupBySchemaFlagName) ? 'objectExplorerGroupBySchemaEnabled' : 'objectExplorerGroupBySchemaDisabled';
+			TelemetryReporter.sendActionEvent('mssql.objectExplorer', groupBySchemaTelemetryActionEvent);
```

Comment:
I would suggest making a const enum for both action and view names like we do elsewhere (e.g. https://github.com/Microsoft/azuredatastudio/blob/60c1c8a89f64d02d2777e074d75bc917f23db97b/extensions/dacpac/src/telemetry.ts). It helps easily track down what different values we can expect